code/modules/clothing/suits/armor.dm

```diff
@@ -458,8 +458,8 @@
 	icon_state = "ert_soldier"
 	item_state = "ert_soldier"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
-	armor = list(melee = 60, bullet = 60, laser = 60, energy = 40, bomb = 20, bio = 0, rad = 0)
-	slowdown = 0
+	armor = list(melee = 65, bullet = 60, laser = 60, energy = 40, bomb = 20, bio = 25, rad = 25)
+	slowdown = 1

```

Comment:
Patently false.
The heavy armor vests mercs get has armor = list(melee = 60, bullet = 60, laser = 60, energy = 40, bomb = 40, bio = 25, rad = 25). This is 5 less melee armor than the ERT vest. Any more, and it would be better than tactical armor.

Reply:
You forgot to count the bombprotection

Reply:
Yes. The ERT chestpieces also cover more of the body.

Reply:
This statement is incorrect.
The ERT armor vest covers: UPPER_TORSO, LOWER_TROSO, ARM_LEFT, ARM_RIGHT
The merc armor covers: UPPER_TORSO, LOWER_TORSO, ARM_LEFT, ARM_RIGHT, LEG_LEFT, LEG_RIGHT